requirements/requirements_docs.txt

```diff
@@ -2,3 +2,4 @@ latexcodec
 sphinx_rtd_theme
 sphinxcontrib-bibtex
 wrapt
+megatron-lm==1.1.4
```

Comment:
This requirement file should never be changed. If you need a module mock it here: https://github.com/NVIDIA/NeMo/blob/main/docs/source/conf.py#L31-L72